contracts/PvpArena.sol

```diff
@@ -201,6 +203,7 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         // TODO: Tweak these values, they are placeholders
         wageringFactor = 3;
         reRollFeePercent = 25;
+        withdrawFeePercent = 25;
```

Comment:
just a heads up this is not going to be initialized on the testnets we've already deployed on

Reply:
We added a setter for this value so you can call in the new migration file.